ledger/src/shred/merkle.rs

```diff
@@ -148,12 +150,14 @@ impl ShredData {
     //   ShredCode::capacity(proof_size).unwrap()
```

Comment:
nit: 
```suggestion
    //   ShredCode::capacity(proof_size, chained, resigned).unwrap()
```

Reply:
also for consistency update the comment in `impl ShredTrait for ShredData`

Reply:
done.